src/libs/ReportSecondaryActionUtils.ts

```diff
@@ -338,15 +338,12 @@ function isMoveTransactionAction(reportTransactions: Transaction[], reportAction
     return canMoveExpense;
 }
 
-function isDeleteAction(report: Report, reportTransactions: Transaction[], reportActions?: ReportAction[]): boolean {
+function isDeleteAction(report: Report, reportActions?: ReportAction[]): boolean {
     const transactionThreadReportID = getOneTransactionThreadReportID(report.reportID, reportActions ?? []);
     const isExpenseReport = isExpenseReportUtils(report);
     const isIOUReport = isIOUReportUtils(report);
 
-    // This should be removed when is merged https://github.com/Expensify/App/pull/58020
-    const isSingleTransaction = reportTransactions.length === 1;
-
-    if ((!isExpenseReport && !isIOUReport) || !isSingleTransaction || (!!reportActions && !transactionThreadReportID)) {
+    if ((!isExpenseReport && !isIOUReport) || (!!reportActions && !transactionThreadReportID)) {
```

Comment:
I think the check for the transactionThreadReportID is incorrect for reports.
```suggestion
    if ((!isExpenseReport && !isIOUReport)) {
```